app/src/main/java/com/zeapo/pwdstore/autofill/v2/AutofillUtils.kt

```diff
@@ -0,0 +1,41 @@
+package com.zeapo.pwdstore.autofill.v2
+
+import android.content.Context
+import android.os.Build
+import android.service.autofill.Dataset
+import android.widget.RemoteViews
+import androidx.annotation.RequiresApi
+import com.zeapo.pwdstore.R
+import android.view.autofill.AutofillValue
+
+
+@RequiresApi(api = Build.VERSION_CODES.O)
+class AutofillUtils {
```

Comment:
Make this an object and drop the `companion object` inside.

Reply:
For my taste it's better to make extension functions for `Context` instead of object.

Reply:
Each function is used only once outside this fuke, I'd much rather have them in a `Utils` object than an extension function.